.github/workflows/ci.yml

```
@@ -0,0 +1,40 @@
+      - name: Install dependencies
+        run: pnpm install
+
+      - name: Run lint
+        run: pnpm lint
+
+      - name: Run format check
+        run: pnpm format:check
+
+  integration-tests:
+    needs: lint-and-format
+    uses: ./.github/workflows/integration-tests.yml
```

actionlint (1.7.10)

[error] error while parsing reusable workflow "./.github/workflows/integration-tests.yml": "workflow_call" event trigger is not found in "on:" at line:4, column:3

(workflow-call)